pkg/apis/apps/validation/validation_test.go

```diff
@@ -249,7 +249,7 @@ func TestValidateStatefulSet(t *testing.T) {
 				UpdateStrategy: apps.StatefulSetUpdateStrategy{Type: apps.RollingUpdateStatefulSetStrategyType},
 			},
 		},
-		"invalid udpate strategy": {
+		"invalid update strategy": {
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464. Closing.

/close